src/main/java/com/ait/lienzo/shared/core/types/Color.java

```diff
@@ -136,10 +141,13 @@ else if (percent > 0)
 
     /**
      * Generates a unique RGB color key, e.g. "rgb(12,34,255)".
-     * This is used internally.
-     * 
+     *
+     * @Deprecated
```

Comment:
I can't find out where this method is used, but it seems like Internal method but with public modifier. So I mark it as Depricated and don't introduce tests for it.